src/lv_core/lv_obj.h

```diff
@@ -102,6 +102,7 @@ enum {
     LV_EVENT_APPLY,  /**< "Ok", "Apply" or similar specific button has clicked*/
     LV_EVENT_CANCEL, /**< "Close", "Cancel" or similar specific button has clicked*/
     LV_EVENT_DELETE, /**< Object is being deleted */
+    _LV_EVENT_LAST /** Number of events*/
```

Comment:
Probably convenient, but never actually referenced?

Reply:
Oh... I was using this when I was testing adding custom widget level events. Definitely should be removed given that event extension is not currently something that is needed.